### PR TITLE
Add jQueryFormStyler@2.0.2 w/ git auto-update

### LINKED ISSUE #9300: [Request] Add jquery.formstyler
**Library name:** jquery.formstyler
**Git repository url:** https://github.com/Dimox/jQueryFormStyler
**npm package url(optional):** 
**License(s):** MIT https://github.com/Dimox/jQueryFormStyler/blob/master/MIT-LICENSE.md
**Official homepage:** http://dimox.name/jquery-form-styler/
**Wanna say something? Leave message here:**
# 

Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by sashberd**
close #9300, cc @Dimox

Pull request for issue: #9300
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/Dimox/jQueryFormStyler
 * Official website (optional, not the repository): http://dimox.name/jquery-form-styler/
 * NPM package url (optional): https://www.npmjs.com/package/jquery.formstyler
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 34
   - Count of stars:222
   - Count of forks:101
 * NPM download stats (optional):
   - Downloads in the last day: 1
   - Downloads in the last week: 46
   - Downloads in the last month: 418

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by Dimox** [comment]
The correct NPM package url - https://www.npmjs.com/package/jquery-form-styler

**Comment by sashberd** [comment]
@Dimox Thanks for your feedback. The PR info was updated

**Comment by sufuf3** [comment]
@extend1994 @maruilian11 Please help review this PR, thanks.

File: ajax/libs/jquery.formstyler/package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "jquery.formstyler",
+  "version": "2.0.0",
+  "description": "jQuery-плагин для стилизации элементов HTML-форм",
```
[the remark this answers is not in the document]
Comment (extend1994) [reply ajax/libs/jquery.formstyler/package.json]: Note: v2.0.1 is only on npm.

[the remark this answers is not in the document]
Comment (sashberd) [reply ajax/libs/jquery.formstyler/package.json]: I can translate it from Russian (my native) to English... WDYT?

[the remark this answers is not in the document]
Comment (sashberd) [reply ajax/libs/jquery.formstyler/package.json]: v2.0.1 was manually added from npm in next PR. FYR

[the remark this answers is not in the document]
Comment (extend1994) [reply ajax/libs/jquery.formstyler/package.json]: @sashberd It would be great if you can do the translation :smiley:

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

File: ajax/libs/jquery.formstyler/package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "jquery.formstyler",
```
Comment (extend1994) [inline ajax/libs/jquery.formstyler/package.json]: Let's use git repo name `jQueryFormStyler` as this library's name. Sorry for not noticing this previously.

**Comment by sashberd** [comment]
@extend1994 
I readd all files in all versions in both commits via WGET tool
I reminified them with  --no-map option
I rename the library to name that you are mention in you last review
If the problem with checksum from you side is still existed, I have no idea how to fix. Only if you will add those files from you side

**Comment by sashberd** [comment]
@extend1994 ping

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

**Pull request merged**